electronics/src/Config.h

```diff
@@ -0,0 +1,103 @@
+#ifndef CONFIG_H
+#define CONFIG_H
+
+#include "Handles.h"
+#include <PinNames.h>
+
+/**
+ *
+ * -----------------------------------------------
+ * SET MODE [Type of controller to be deployed to]
+ * -----------------------------------------------
+ * 
+ */
+
+// #define RAM // Red Alliance controller mode
```

Comment:
"RAM" could be misconstrued as Random Access Memory instead of Red Alliance Mode - maybe another name is a better idea

Reply:
I'll think of a new name